src/components/Editor/Invitees/OrganizerList.vue

```diff
@@ -0,0 +1,81 @@
+<!--
+  - SPDX-FileCopyrightText: 2019 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<template>
+	<NcSelect class="organizer-select__multiselect"
+		:options="organizerSelection"
+		:searchable="false"
+		:max-height="600"
+		:placeholder="placeholder"
+		:class="{ 'showContent': inputGiven, 'icon-loading': isLoading }"
+		:clearable="false"
+		:label-outside="true"
+		:value="organizerSelected"
+		input-id="organizer"
+		label="label"
+		@option:selected="changeOrganizer">
+		<template #option="option">
+			<div class="organizer-select-list-item">
+				<!-- We need to specify a unique key here for the avatar to be reactive. -->
+				<Avatar :key="option.address"
+					:user="option.avatar"
+					:display-name="option.label" />
+				<div class="organizer-select-list-item__label">
+					<div>
+						{{ option.label }}
+					</div>
+					<div v-if="option.address !== option.label">
+						{{ option.address }}
+					</div>
+				</div>
+			</div>
+		</template>
+	</NcSelect>
+</template>
+
+<script>
+import {
+	NcAvatar as Avatar,
+	NcSelect,
+} from '@nextcloud/vue'
+
+export default {
+	name: 'OrganizerList',
+	components: {
+		Avatar,
+		NcSelect,
+	},
+	props: {
+		organizerSelection: {
+			type: Array,
+			required: true,
+		},
+		organizerSelected: {
+			type: Object,
+			required: false,
+		},
+	},
+	data() {
+		return {
+			isLoading: false,
+			inputGiven: false,
+			matches: [],
+		}
+	},
+	computed: {
+		placeholder() {
+			return this.$t('calendar', 'Select the Organizer of this event')
+		},
+		noResult() {
+			return this.$t('calendar', 'No match found')
+		},
+	},
+	methods: {
+		changeOrganizer(selectedValue) {
+			this.$emit('change-organizer', selectedValue)
```

Comment:
```suggestion
			this.$emit('update:value', selectedValue)
```

Again, this is a convention in the vue world.